### PR TITLE
feat: [CDS-80632]: Remove extra logs to reduce log cost on prod

### PR DESCRIPTION
**Pull request by Y0709**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/54101)

### CODE REVIEW

File: pipeline-service/modules/orchestration-visualization/src/main/java/io/harness/service/impl/GraphGenerationServiceImpl.java [hunk 1]
```diff
@@ -226,8 +226,11 @@ boolean updateGraphUnderLock(OrchestrationGraph orchestrationGraph) {
       executionSummaryUpdate.set(PlanExecutionSummaryKeys.lastUpdatedAt, lastUpdatedAt);
       pmsExecutionSummaryService.update(planExecutionId, executionSummaryUpdate);
     }
-    log.info("[PMS_GRAPH] Processing of [{}] orchestration event logs completed in [{}ms]", unprocessedEventLogs.size(),
-        System.currentTimeMillis() - startTs);
+    long diff = System.currentTimeMillis() - startTs;
+    if (diff > 1000) {
```
Comment (sahilhindwani-harness) [inline pipeline-service/modules/orchestration-visualization/src/main/java/io/harness/service/impl/GraphGenerationServiceImpl.java:230]: nit: lets use constant for this.

**Review by archit-harness — CHANGES_REQUESTED** [review]
> Reviewed 5 of 13 files at r1, 5 of 6 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/54101)* status: 13 file(s) to review. 1 change requests. 0 of 2 approvals obtained. 

___
*[`884-pms-commons/src/main/java/io/harness/pms/events/base/PmsAbstractRedisConsumer.java` line 133 at r2](https://reviewable.io/reviews/harness/harness-core/54101#-Ngb3tvj1AZzyNuckuTK:-Ngb3tvj1AZzyNuckuTL:bfexm0b) ([raw file](https://github.com/harness/harness-core/blob/76bd66f9d7bfce1fac521acce4adbe7673d1cee0/884-pms-commons/src/main/java/io/harness/pms/events/base/PmsAbstractRedisConsumer.java#L133)):*
> ```Java
>     }
>     List<Message> notProcessableMessages =
>         messages.stream().filter(message -> !processableMessages.contains(message)).collect(Collectors.toList());
> ```

can we go message.getId compare?



_Code quote:_
```Java
!processableMessages.contains(message)
```
___
*[`pipeline-service/modules/orchestration/src/main/java/io/harness/engine/pms/commons/events/PmsEventSender.java` line 107 at r2](https://reviewable.io/reviews/harness/harness-core/54101#-Ngb54K0EJoPn8eOxVo3:-Ngb54K0EJoPn8eOxVo4:bp1imsf) ([raw file](https://github.com/harness/harness-core/blob/76bd66f9d7bfce1fac521acce4adbe7673d1cee0/pipeline-service/modules/orchestration/src/main/java/io/harness/engine/pms/commons/events/PmsEventSender.java#L107)):*
> ```Java
>         producer.send(Message.newBuilder().putAllMetadata(metadataBuilder.build()).setData(eventData).build());
>     log.info("Successfully Sent {} event for {} to the producer. MessageId {} in [{}ms]", eventCategory, serviceName,
>         messageId, System.currentTimeMillis() - startTs);
> ```

You can probably use - ResponseTimeRecorder
___
*[`pipeline-service/modules/orchestration/src/main/java/io/harness/engine/pms/execution/strategy/AbstractNodeExecutionStrategy.java` line 98 at r2](https://reviewable.io/reviews/harness/harness-core/54101#-Ngb5K2T3wRKvmrcO6U7:-Ngb5K2T3wRKvmrcO6U8:b83ndyd) ([raw file](https://github.com/harness/harness-core/blob/04d3e998d7d136c3aff5d6edb438e6e29ecc9b74/pipeline-service/modules/orchestration/src/main/java/io/harness/engine/pms/execution/strategy/AbstractNodeExecutionStrategy.java#L98)):*
> ```Java
>       SdkResponseProcessor handler = sdkResponseProcessorFactory.getHandler(event.getSdkResponseEventType());
>       handler.handleEvent(event);
>       log.info("Event for SdkResponseEvent for event type {} completed successfully", event.getSdkResponseEventType());
> ```

Did we decide this? i think we mentioned we will keep success one with time as of now

File: pipeline-service/modules/orchestration-visualization/src/main/java/io/harness/service/impl/GraphGenerationServiceImpl.java (hunk 1 above)
Comment (Y0709) [reply pipeline-service/modules/orchestration-visualization/src/main/java/io/harness/service/impl/GraphGenerationServiceImpl.java]: done

**Review by Y0709 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/54101)* status: 13 file(s) to review. 1 change requests. 2. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`884-pms-commons/src/main/java/io/harness/pms/events/base/PmsAbstractRedisConsumer.java` line 133 at r2](https://reviewable.io/reviews/harness/harness-core/54101#-Ngb3tvj1AZzyNuckuTK:-NgrbY5zDvbVMdWu5Z9T:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/76bd66f9d7bfce1fac521acce4adbe7673d1cee0/884-pms-commons/src/main/java/io/harness/pms/events/base/PmsAbstractRedisConsumer.java#L133)):*
<details><summary><i>Previously, archit-harness (Archit Singla) wrote…</i></summary><blockquote>

can we go message.getId compare?
</blockquote></details>

Done.
___
*[`pipeline-service/modules/orchestration/src/main/java/io/harness/engine/pms/execution/strategy/AbstractNodeExecutionStrategy.java` line 98 at r2](https://reviewable.io/reviews/harness/harness-core/54101#-Ngb5K2T3wRKvmrcO6U7:-NgrbaLQ8BGgvWqdChkS:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/04d3e998d7d136c3aff5d6edb438e6e29ecc9b74/pipeline-service/modules/orchestration/src/main/java/io/harness/engine/pms/execution/strategy/AbstractNodeExecutionStrategy.java#L98)):*
<details><summary><i>Previously, archit-harness (Archit Singla) wrote…</i></summary><blockquote>

Did we decide this? i think we mentioned we will keep success one with time as of now
</blockquote></details>

Done.

**Comment by Y0709** [comment]
trigger uts

**Review by archit-harness — APPROVED** [review]
> Reviewed 1 of 13 files at r1, 3 of 3 files at r3, 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/54101)* status: 13 file(s) to review. 1 of 2 approvals obtained. 

___
*[`pipeline-service/modules/orchestration/src/main/java/io/harness/engine/pms/commons/events/PmsEventSender.java` line 107 at r2](https://reviewable.io/reviews/harness/harness-core/54101#-Ngb54K0EJoPn8eOxVo3:-Ngwg8mn85Sci7-1XciG:bb7f2mf) ([raw file](https://github.com/harness/harness-core/blob/76bd66f9d7bfce1fac521acce4adbe7673d1cee0/pipeline-service/modules/orchestration/src/main/java/io/harness/engine/pms/commons/events/PmsEventSender.java#L107)):*
<details><summary><i>Previously, archit-harness (Archit Singla) wrote…</i></summary><blockquote>

You can probably use - ResponseTimeRecorder
</blockquote></details>

done

**Comment by Y0709** [comment]
trigger messagecheck, review

**Pull request merged**